pkg/asset/installconfig/gcp/gcp.go

```diff
@@ -38,16 +38,48 @@ func selectProject(ctx context.Context) (string, error) {
 	}
 	defaultProject := ssn.Credentials.ProjectID
 
+	client := &Client{
+		ssn: ssn,
+	}
+
+	var listOfCombo []string
+
+	listOfNames, listOfIds, err := client.GetListOfProjects(ctx)
+	if err != nil {
+		return "", err
+	}
+	index := 0
+	for i, v := range listOfIds {
+		if v == defaultProject {
+			index = i
+		}
+		listOfCombo = append(listOfCombo, listOfNames[i]+" ("+v+")")
```

Comment:
the string formatting here should use `fmt.Sprintf`. Also I think it's more readable to have the string formatting in its own line and assigned to a variable, rather than as an arg to `append`.